pkg/apis/istio/v1alpha3/virtual_service.go

```diff
@@ -725,7 +725,7 @@ type HTTPRetry struct {
 	// REQUIRED. Number of retries for a given request. The interval
 	// between retries will be determined automatically (25ms+). Actual
 	// number of retries attempted depends on the httpReqTimeout.
-	Attempts int `json:"attempts,omitempty"`
+	Attempts int `json:"attempts"`
```

Comment:
Can you please undo this as it's not related to this PR. We can do this in a different PR.

Reply:
This change was not part of this PR.
I merged the master branch into this one, and now it doesn't show this diff anymore.